textsubs.lic

```diff
@@ -890,4 +890,302 @@ TextSubs.add('Your skill in (.*) is at a zenith of enlightenment\.', 'Your skill
 # Commas when dealing with coins
 TextSubs.add('\d{1,3}(?=(\d{3})+(?!\d).*(Kronar|Dokora|Lirum))', '\&,')
 
+# Scroll subs to add guild name
+TextSubs.add('"Strange Arrow."', '"Strange Arrow(Analagous)."')
+TextSubs.add('"Burden."', '"Burden(Analagous)."')
+TextSubs.add('"Lay Ward."', '"Lay Ward(Analagous)."')
+TextSubs.add('"Manifest Force."', '"Manifest Force(Analagous)."')
+TextSubs.add('"Ease Burden."', '"Ease Burden(Analagous)."')
```

Comment:
I'd personally leave the period in the quotation marks and just have the spell book separate outside in the ( ), but whatever most people prefer is fine with me.